pdc/apps/module/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2015 Red Hat
+# Copyright (c) 2018 Red Hat
```

Comment:
I'll make these changes in a separate commit. I made the change in this PR because I felt like the year should be updated since this basically felt like a rewrite of the `module` Django app. If you disagree, I can get rid of the commit.